### PR TITLE
[#5] Add peer roles implementation

### LINKED ISSUE #5: Add peer roles implementation
1- committer role
2- endorser role
3- validator role

### PR DESCRIPTION
**Pull request by fqutishat**
Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

closes #5

### CODE REVIEW

File: mod/peer/roles/ledger_roles_config_test.go [hunk 1]
```diff
@@ -13,15 +13,15 @@ import (
 )
 
 func TestIsCommitter(t *testing.T) {
-	require.True(t, IsCommitter())
+	require.NotNil(t, IsCommitter())
```
Comment (bstasyszyn) [inline mod/peer/roles/ledger_roles_config_test.go]: I don't understand NotNil for a bool?

Comment (fqutishat) [reply mod/peer/roles/ledger_roles_config_test.go]: done

**Comment by fqutishat** [comment]
verify-unitest

**Comment by fqutishat** [comment]
verify-unitest

**Pull request merged**